.gitignore

```diff
@@ -1,3 +1,4 @@
 /bin/
 coverage.txt
 profile.out
+.vscode
```

Comment:
oops. I think we should remove this line~

Reply:
sure .. removed

never sure of when it's ok to add .gitignore paths / files   just removing an eye sore from my git status check ...

I'll figure out why vsc started adding those maybe I can config it out vs ignore it :-)